students/franjaku/Lesson_3/mailroom.py

```diff
@@ -0,0 +1,166 @@
+import sys
+#Mailroom.py
+
+
+# Database Functions
+"""
+Database requirements
+-keep track of: donor names, total amount donated, number of times donated
+                average donation.
+
+Database Structure
+    [('Donor Name', Total donated, number dontations, average donation),
+     ('Donor Name', Total donated, number dontations, average donation),
+     ...]
+
+"""
+
+def initialize_database():
+    database = [('John Smith',5000, 1, 5000),
+                ('Jane Adams',25000,1, 25000),
+                ('Brett Johnson', 50, 1, 50),
+                ('Sofia Pippy', 623, 1, 623),
+                ('Maddy North', 85426, 1, 85426)]
+    return database
+
+def print_donor_list(database):
+    print ('\nDonor List\n----------')
+    for donor in database:
+        print('{}'.format(donor[0]))
+    print('----------')
+
+def add_new_donor(database, name,donation_amount):
+    new_donor = (name,donation_amount,1,donation_amount)
+    database.append(new_donor)
+    return database
+
+def add_new_donation(database,name,donation_amount):
+    #by default this is for existing donors
+    donor_found=0
+
+    for idx,donor in enumerate(database):
+        if donor[0] == name:
+            donor_found=1
+
+            #update donor data
+            total_donation_new= donor[1] + donation_amount
+            number_donations_new = donor[2] + 1
+            average_donation_new = total_donation_new/number_donations_new
+            donor_updated = (name,total_donation_new,number_donations_new,
+                            average_donation_new)
+
+            database[idx] = donor_updated
```

Comment:
you can use an `else` statement with for loop instead of `donor_found` check to simplify your code.